client/verta/tests/test_deployment.py

```diff
@@ -1025,3 +1021,31 @@ def test_undeployed_get_error(self, experiment_run, model_for_deployment):
             "{}://{}/api/v1/deployment/models/{}".format(conn.scheme, conn.socket, experiment_run.id),
             headers=conn.auth,
         )
+
+
+class TestGitOps:
+    def test_download_deployment_crd(self, experiment_run, model_for_deployment, in_tempdir):
+        download_to_path = "deployment.yaml"
+
+        experiment_run.log_model(
+            model_for_deployment['model'],
+            custom_modules=[],
+            model_api=model_for_deployment['model_api'],
+        )
+        experiment_run.log_requirements(['scikit-learn'])
+
+        filepath = experiment_run.download_deployment_crd(download_to_path)
+        assert filepath == os.path.abspath(download_to_path)
+
+        # can be loaded as YAML
+        with open(filepath, 'rb') as f:
+            model_deployment, config_map = yaml.safe_load_all(f)
+
+        assert model_deployment['kind'] == "ModelDeployment"
+        assert model_deployment['metadata']['name'] == experiment_run.id
+
+        assert config_map['kind'] == "ConfigMap"
+        assert config_map['metadata']['name'] == "model--{}".format(experiment_run.id)
+
+        model_api = json.loads(config_map['data']['model_api.json'])
+        assert model_api == model_for_deployment['model_api'].to_dict()
```

Comment:
@conradoverta Is there anything else I should cover in this test?

Reply:
No. We'll have to remember to change the test as things goes.